server/settings/publish.py

```diff
@@ -235,6 +238,11 @@ class PublishPluginsModel(BaseSettingsModel):
         "optional": True,
         "active": True
     },
+    "ValidateSingleFrame": {
+        "enabled": True,
+        "optional": True,
```

Comment:
Default should be false here because this validator runs on instances that should _really_ be static by default.
```suggestion
        "optional": False,
```